all-actors/src/main/java/org/sunbird/CertVars.java

```diff
@@ -14,7 +14,7 @@
     private  static final String SERVICE_BASE_URL=getPropsFromEnvs(CERT_SERVICE_BASE_URL);
     private static final String DOWNLOAD_URI= "/v1/user/certs/download";
```

Comment:
why use user here? Remove that too.

Reply:
@indrajra cert-service has `user` keyword in there URI. so need to change **URI** in cert-service, you can check [here](https://github.com/project-sunbird/cert-service/blob/master/service/conf/routes)